controllers/account/api/v1/debt_types.go

```diff
@@ -27,18 +27,17 @@ const (
 	DebtStatusLarge  DebtStatusType = "Large"
 	DebtPrefix                      = "debt-"
 	NormalPrice                     = 0
-	// SmallBlockWaitSecond 3 days
-	SmallBlockWaitSecond = 3 * 24 * 60 * 60
-	// MediumBlockWaitSecond 4 days
-	MediumBlockWaitSecond = 4 * 24 * 60 * 60
+	// SmallBlockWaitSecond 7 days
+	SmallBlockWaitSecond = 7 * 24 * 60 * 60
```

Comment:
Is the `BlockWaitSecond` here be the only location to control delete timeout? Witch measn if we wangto change it to 30days only need to modify here one place.

Reply:
no,I have  answered in the next comment.